### PR TITLE
Spellchecking

### PR DESCRIPTION
**Pull request by sfc-gh-chu**
# Description

Some spellcheck corrections. Found with codespell

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by sfc-gh-jreini — APPROVED** [review]

**Comment by sfc-gh-jreini** [comment]
How is codespell run/is this checked in tests somehow?

**Comment by sfc-gh-chu** [comment]
> How is codespell run/is this checked in tests somehow? Or is it a one-time thing

There's a good amount of false positives from codespell so didn't include this in testing. Will need some more time to clean up exemptions so this is just a first pass and one time thing

File: src/benchmark/trulens/benchmark/groundedness_benchmark.ipynb [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
+    "SummEval is one of the datasets dedicated to automated evaluations on summarization tasks, which are closely related to the groundedness evaluation in RAG with the retrieved context (i.e. the source) and response (i.e. the summary). It contains human annotation of numerical score (**1** to **5**) comprised of scoring from 3 human expert annotators and 5 crowed-sourced annotators. There are 16 models being used for generation in total for 100 paragraphs in the test set, so there are a total of 16,000 machine-generated summaries. Each paragraph also has several human-written summaries for comparative analysis. \n",
```
Comment (sfc-gh-pdharmana) [inline src/benchmark/trulens/benchmark/groundedness_benchmark.ipynb:13]: This is crowd?

File: src/benchmark/trulens/benchmark/groundedness_benchmark_snowflake_arctic.ipynb [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
+    "SummEval is one of the datasets dedicated to automated evaluations on summarization tasks, which are closely related to the groundedness evaluation in RAG with the retrieved context (i.e. the source) and response (i.e. the summary). It contains human annotation of numerical score (**1** to **5**) comprised of scoring from 3 human expert annotators and 5 crowed-sourced annotators. There are 16 models being used for generation in total for 100 paragraphs in the test set, so there are a total of 16,000 machine-generated summaries. Each paragraph also has several human-written summaries for comparative analysis. \n",
```
Comment (sfc-gh-pdharmana) [inline src/benchmark/trulens/benchmark/groundedness_benchmark_snowflake_arctic.ipynb:13]: Same here? crowd?

**Review by sfc-gh-pdharmana — COMMENTED** [review]
Why did poetry.lock files change?

**Comment by sfc-gh-chu** [comment]
> Why did poetry.lock files change?

There were some conflicts on the lock files so just recompiled them

**Pull request merged**